itest/utils.go

```diff
@@ -20,7 +22,9 @@ import (
 )
 
 var (
-	zeroHash chainhash.Hash
+	zeroHash          chainhash.Hash
+	regtestMiningAddr = "n1VgRjYDzJT2TV72PnungWgWu18SWorXZS"
```

Comment:
Is seed generation by the nodes deterministic? 

Do we need to also move to simnet, so bitcoind won't wipe the chain on start up?

Reply:
This is just to send to `bitcoind` to specify _where_ to mine the the coins to. We don't really ever use that address for anything else.

No, it's `btcd` that drops the chain on startup, not `bitcoind`. So we can switch batch to `bitcoind` and `regtest`.